lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -66,16 +66,18 @@ def define
         end
 
         next_node do
-          if calculator.ceremony_country == 'monaco'
-            if calculator.want_to_get_married?
+          if calculator.want_to_get_married?
+            if calculator.ceremony_country == 'monaco'
               outcome :outcome_marriage_in_monaco
             else
```

Comment:
This question (`marriage_or_pacs?`) can only be reached if the ceremony country offers [PACS](https://en.wikipedia.org/wiki/Civil_solidarity_pact). According to our system, only France, Monaco, New Caledonia and Wallis and Futuna offer PACS. So, if the ceremony country isn't Monaco (which is a special case for reasons I don't know) then it must be France or one of the French overseas territories.